app/models/middleware_messaging.rb

```diff
@@ -3,4 +3,22 @@ class MiddlewareMessaging < ApplicationRecord
   belongs_to :middleware_server, :foreign_key => "server_id"
   acts_as_miq_taggable
   serialize :properties
+
+  include LiveMetricsMixin
+
+  def metrics_capture
+    @metrics_capture ||= ManageIQ::Providers::Hawkular::MiddlewareManager::LiveMetricsCapture.new(self)
+  end
+
+  def live_metrics_name
```

Comment:
In the case of middleware domain, all metrics are based on LiveMetricsMixin.
Yes, I mean only for this case.